src/core/App.js

```diff
@@ -17,15 +14,18 @@ import ImageUploader from '../common/components/ImageUploader/ImageUploader'
 class App extends Component {
   constructor () {
     super()
-    this.state = {}
-  }
-  fetchData (query, random) {
-  }
-  setQuery () {
+    this.state = {
+      firstCaption: {},
+      secondCaption: {}
+    }
 
+    this.getResponse = this.getResponse.bind(this)
+  }
+  getResponse (obj) {
+    this.setState({ firstCaption: obj.BestOutcome })
+    this.setState({ secondCaption: obj.RunnerUpOutcome })
```

Comment:
is this mutating state? 🤔

Reply:
No, but it is more conventional to combine the setState, e.g.:
```
this.setState({ 
     firstCaption: obj.BestOutcome,
     secondCaption: obj.RunnerUpOutcome
})
```

Reply:
Sankyuuu for the suggestion @sarahhamdi !